### PR TITLE
Added sidebar onclick functionality for mobile

### PR DESCRIPTION
**Pull request by y-arjun-y**
Made it so that the sidebar collapses when clicked (itself or the links) on mobile.

Note: I added a new pip package, so before merging, should I do anything with the pipfile?)

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -34,17 +36,31 @@
...
+    if x: 
+        return "document.getElementById('sidebar').setAttribute('onclick', 'mobileSidebar()')"
+    return ""
+
 # app.layout = serve_layout
 
 if __name__ == '__main__':
     #app.run_server(debug=True)
-    app.run_server(debug=False)
+    app.run_server(host="0.0.0.0", debug=True)
```
Comment (hamishhuggard) [inline app.py:66]: Why the change in IP address?

File: assets/main.js [hunk 2]
```diff
@@ -26,6 +26,15 @@ function toggleAboutVisibility() {
 }
 
 
+if (window.matchMedia("(max-media: 700px)").matches) {
```
Comment (hamishhuggard) [inline assets/main.js:29]: It would make more sense if the conditional was inside the function, no?
Otherwise if the function is called from desktop we'll get a "function not defined" type error.

File: app.py (hunk 1 above)
Comment (y-arjun-y) [reply app.py:66]: Yeah, this is a mistake. I originally did it so I can see the app on mobile. I'll change it.

File: assets/main.js (hunk 2 above)
Comment (y-arjun-y) [reply assets/main.js:29]: I think I remember putting it inside the function and it not working. But, I'll try it inside the function and get back to you.

Comment (y-arjun-y) [reply assets/main.js:29]: Yeah, it doesn't work if the conditional is inside the function.
